blockchain/chain_sync/src/sync.rs

```diff
@@ -987,8 +1013,12 @@ mod tests {
             Cid::from_raw_cid("bafy2bzacecujyfvb74s7xxnlajidxpgcpk6abyatk62dlhgq6gcob3iixhgom")
                 .unwrap();
 
-        let root =
-            ChainSyncer::compute_msg_data(cs.chain_store.blockstore(), &[bls], &[secp]).unwrap();
+        let root = ChainSyncer::<MemoryDB, MockBeacon>::compute_msg_data(
```

Comment:
Seems weird for this to be associated, you shouldn't need to infer the beacon type for this. At very least, you shouldn't have to specify DB generic here. Probably just makes sense to move this fn out of the ChainSyncer impl
```suggestion
        let root = ChainSyncer::<_, MockBeacon>::compute_msg_data(
```

Reply:
Agree. I'm taking it out of the impl block